hal/targets/hal/TARGET_STM/TARGET_STM32L4/rtc_api.c

```diff
@@ -39,11 +40,29 @@ static int rtc_inited = 0;
 
 static RTC_HandleTypeDef RtcHandle;
 
+#if DEVICE_RTC_LSI
```

Comment:
can we reserve DEVICE_ for device_has magic list. If a macro just use RTC_LSI or STM_RTC_LSI